src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -102,6 +105,12 @@ public void AddNodeStats(StringBuilder benchLogs)
             }
         }
 
+        /// <inheritdoc />
+        public override void LoadConfiguration()
+        {
+            this.consensusSettings.LoadFromConfig();
```

Comment:
Can we have a nodeSettings here and a Load() method in the ConsensusSettings so that the calls to load the settings look the same in all the features?

Reply:
Done.